openforcefield/typing/engines/smirnoff/forcefield.py

```diff
@@ -885,8 +885,10 @@ def _load_smirnoff_data(self, smirnoff_data, allow_cosmetic_attributes=False):
                 parameter_list_dict = {}
             else:
                 parameter_list_dict = section_dict.pop(parameter_list_tagname, {})
-                # Must be wrapped into its own tag.
-                parameter_list_dict = {parameter_list_tagname: parameter_list_dict}
+
+                # If the parameter list isn't empty, it must be transferred into its own tag.
+                if parameter_list_dict != {}:
```

Comment:
This is necessary for deserializing SMIRNOFF force field sections which may or may not have any smirks-based elements (like an empty `ChargeIncrementModel` tag)